code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -4364,3 +4407,64 @@
 	New()
 		..()
 		reagents.add_reagent(NUTRIMENT,3)
+
+/obj/item/weapon/reagent_containers/food/snacks/bait
+	name = "bait"
+	desc = "Don't fall for it."
+	icon_state = "bacon" //placeholder
+	bitesize = 1
+	New()
+		..()
+		reagents.add_reagent(NUTRIMENT,1)
+
+/obj/item/weapon/reagent_containers/food/snacks/fish
+	name = "raw fish"
+	desc = "The product of a real man who lives off the land."
+	icon_state = "meat" //placeholder
+	bitesize = 1
+	var/list/fish_guts = list(/obj/item/weapon/reagent_containers/food/snacks/meat/fish_fillet) // contains anything the fish will will drop when filleted
+	New()
+		..()
+		reagents.add_reagent(NUTRIMENT,1)
+
+/obj/item/weapon/reagent_containers/food/snacks/fish/attackby(obj/item/weapon/W, mob/user)
+	if(istype(W, /obj/item/weapon/kitchen/utensil/knife/large))
+		Fillet(user)
+	else
+		..()
+
+/obj/item/weapon/reagent_containers/food/snacks/fish/proc/Fillet(mob/user)
+	if(!isturf(loc) && !user.is_holding_item(src))
+		return
+	var/ground = get_turf(src)
+	if(fish_guts.len == 1) // put fish_guts in hand if there's only 1 and we are filleting in hand
+		var/obj/item/weapon/temp = fish_guts[1]
```

Comment:
Trying to do it that way throws a runtime.  Figured it was a byond problem because `fish_guts[1]` is definitely a typepath:

`[12:04:00] Runtime in snacks.dm,4449: Cannot create objects of type /list.
  proc name: Fillet (/obj/item/weapon/reagent_containers/food/snacks/fish/proc/Fillet)
  usr: Giovanni Burkett (barneygumball) (/mob/living/carbon/human)
  usr.loc: The floor (263, 250, 1) (/turf/simulated/floor)
  src: the raw fish (/obj/item/weapon/reagent_containers/food/snacks/fish)
  src.loc: null
  call stack:
  the raw fish (/obj/item/weapon/reagent_containers/food/snacks/fish): Fillet(Giovanni Burkett (/mob/living/carbon/human))
  the raw fish (/obj/item/weapon/reagent_containers/food/snacks/fish): attackby(the kitchen knife (/obj/item/weapon/kitchen/utensil/knife/large), Giovanni Burkett (/mob/living/carbon/human), "icon-x=18;icon-y=13;left=1;scr...")
  Giovanni Burkett (/mob/living/carbon/human): ClickOn(the raw fish (/obj/item/weapon/reagent_containers/food/snacks/fish), "icon-x=18;icon-y=13;left=1;scr...")
  the raw fish (/obj/item/weapon/reagent_containers/food/snacks/fish): Click(null, "mapwindow.map", "icon-x=18;icon-y=13;left=1;scr...")`